HesoyamHospital/Backend/Repository/MySQLRepository/HospitalManagementRepository/MedicineRepository.cs

```diff
@@ -37,6 +37,9 @@ public IEnumerable<Medicine> GetMedicineByIngredient(Ingredient ingredient)
         public Medicine GetMedicineByName(string name)
             => GetAll().SingleOrDefault(med => med.Name == name);
 
+        public IEnumerable<Medicine> GetAllMedicine()
```

Comment:
Every repository already has GetAll() method. Therefore this is a duplicate method.
GetAll() method is implemented in MySQLRepository.cs class file, therefore it's in MedicineRepository.cs aswell.